integrationTests/relayers/testdata/contracts/eth/bridgeBytecode.txt

```diff
@@ -0,0 +1 @@
60009190848201906040850190845b8181101561188257611b24838551611a38565b9284019291840191600101611b11565b600060208284031215611b4657600080fd5b813560ff8116811461133257600080fd5b60208082526023908201527f41636365737320436f6e74726f6c3a2073656e646572206973206e6f7420416460408201526236b4b760e91b606082015260800190565b634e487b7160e01b600052604160045260246000fd5b60405160c0810167ffffffffffffffff81118282101715611bd357611bd3611b9a565b60405290565b604051601f8201601f1916810167ffffffffffffffff81118282101715611c0257611c02611b9a565b604052919050565b600067ffffffffffffffff821115611c2457611c24611b9a565b5060051b60200190565b6000611c41611c3c84611c0a565b611bd9565b80848252602080830192508560051b850136811115611c5f57600080fd5b855b81811015611cf257803567ffffffffffffffff80821115611c825760008081fd5b90880190601f3681840112611c975760008081fd5b823582811115611ca957611ca9611b9a565b611cba818301601f19168801611bd9565b92508083523687828601011115611cd357600091508182fd5b8087850188850137600090830187015250865250938201938201611c61565b50919695505050505050565b8183526000602080850194508260005b85811015611d3c578135611d218161188e565b6001600160a01b031687529582019590820190600101611d0e565b509495945050505050565b81835260006001600160fb1b03831115611d6057600080fd5b8260051b80836020870137939093016020019392505050565b60005b83811015611d94578181015183820152602001611d7c565b50506000910152565b60c081526000611db160c083018c8e611cfe565b8281036020840152611dc4818b8d611cfe565b90508281036040840152611dd981898b611d47565b90508281036060840152611dee818789611d47565b905084608084015282810360a08401528351808252611e14816020840160208801611d79565b601f01601f1916016020019c9b505050505050505050505050565b634e487b7160e01b600052603260045260246000fd5b600060208284031215611e5757600080fd5b8151801515811461133257600080fd5b634e487b7160e01b600052601160045260246000fd5b600060018201611e8f57611e8f611e67565b5060010190565b80516001600160701b0381168114611ead57600080fd5b919050565b805167ffffffffffffffff81168114611ead57600080fd5b600060808284031215611edc57600080fd5b6040516080810181811067ffffffffffffffff82111715611eff57611eff611b9a565b604052611f0b83611e96565b8152611f1960208401611eb2565b6020820152611f2a60408401611eb2565b6040820152606083015161ffff81168114611f4457600080fd5b60608201529392505050565b805160058110611ead57600080fd5b60006020808385031215611f7257600080fd5b825167ffffffffffffffff811115611f8957600080fd5b8301601f81018513611f9a57600080fd5b8051611fa8611c3c82611c0a565b81815260c09182028301840191848201919088841115611fc757600080fd5b938501935b8385101561204f5780858a031215611fe45760008081fd5b611fec611bb0565b611ff586611e96565b8152868601516120048161188e565b81880152604086810151908201526060808701516120218161188e565b908201526080868101519082015260a061203c818801611f50565b9082015283529384019391850191611fcc565b50979650505050505050565b8082018082111561091457610914611e67565b60008351612080818460208801611d79565b9190910191825250602001919050565b8181038181111561091457610914611e67565b634e487b7160e01b600052603160045260246000fd5b60ff818116838216019081111561091457610914611e6756fea164736f6c6343000814000a
```

Comment:
this file contains hexed byte code. Should we rename it to bridgeBytecode.hex?
Valid for all .txt files

Reply:
updated